sdk/freertos_app_cpu0/src/FreeRTOSConfig.h

```diff
@@ -0,0 +1,245 @@
+ * FreeRTOS API DOCUMENTATION AVAILABLE ON THE FreeRTOS.org WEB SITE.
+ *
+ * See http://www.freertos.org/a00110.html
+ *----------------------------------------------------------*/
+
+/*
+ * The FreeRTOS Cortex-A port implements a full interrupt nesting model.
+ *
+ * Interrupts that are assigned a priority at or below
+ * configMAX_API_CALL_INTERRUPT_PRIORITY (which counter-intuitively in the ARM
+ * generic interrupt controller [GIC] means a priority that has a numerical
+ * value above configMAX_API_CALL_INTERRUPT_PRIORITY) can call FreeRTOS safe API
+ * functions and will nest.
+ *
+ * Interrupts that are assigned a priority above
+ * configMAX_API_CALL_INTERRUPT_PRIORITY (which in the GIC means a numerical
+ * value below configMAX_API_CALL_INTERRUPT_PRIORITY) cannot call any FreeRTOS
+ * API functions, will nest, and will not be masked by FreeRTOS critical
+ * sections (although it is necessary for interrupts to be globally disabled
+ * extremely briefly as the interrupt mask is updated in the GIC).
+ *
+ * FreeRTOS functions that can be called from an interrupt are those that end in
+ * "FromISR".  FreeRTOS maintains a separate interrupt safe API to enable
+ * interrupt entry to be shorter, faster, simpler and smaller.
+ *
+ * The Zynq implements 256 unique interrupt priorities.  For the purpose of
+ * setting configMAX_API_CALL_INTERRUPT_PRIORITY 255 represents the lowest
+ * priority.
+ */
+#define configMAX_API_CALL_INTERRUPT_PRIORITY 18
+
+#define configCPU_CLOCK_HZ                      100000000UL
+#define configUSE_PORT_OPTIMISED_TASK_SELECTION 1
+#define configUSE_TICKLESS_IDLE                 0
+#define configTICK_RATE_HZ                      ((TickType_t) 1000)
+#define configPERIPHERAL_CLOCK_HZ               (33333000UL)
+#define configUSE_PREEMPTION                    1
+#define configUSE_IDLE_HOOK                     0 // TODO: For low-power, set 1 and implement vApplicationIdleHook()
+#define configUSE_TICK_HOOK                     0 // TODO: Set as 1 if we want to run function every tick
+#define configMAX_PRIORITIES                    (7)
+#define configTOTAL_HEAP_SIZE                   (125 * 1024)
+#define configMAX_TASK_NAME_LEN                 (10)
+#define configUSE_TRACE_FACILITY                1
+#define configUSE_16_BIT_TICKS                  0
+#define configIDLE_SHOULD_YIELD                 1
+#define configUSE_MUTEXES                       1
+#define configQUEUE_REGISTRY_SIZE               8
+#define configCHECK_FOR_STACK_OVERFLOW          0
+#define configUSE_RECURSIVE_MUTEXES             1
+#define configUSE_MALLOC_FAILED_HOOK            0
+#define configUSE_APPLICATION_TASK_TAG          0
+#define configUSE_COUNTING_SEMAPHORES           1
+#define configUSE_QUEUE_SETS                    1
+#define configSUPPORT_DYNAMIC_ALLOCATION        1
+#define configSUPPORT_STATIC_ALLOCATION         1
+
+/* WEMPEC ADDITION: Need to specify this callback config so we can create message buffers with callbacks

```

Comment:
"AMDC" ADDITION

Reply:
Shorter comment line length

Reply:
Nothing to do with the fact that the AMDC is losing its WEMPEC exclusivity, huh? 😉